systemtests/scripts/test_func.sh

```diff
@@ -390,20 +390,11 @@ function is_upgraded() {
     TAG=${TAG:-"latest"}
 
     IMAGE_TAG=${IMAGE##*:}
-    TEMPLATES=$(find ${CURDIR}/../../templates/build/enmasse-${TAG} -name '*.yaml' -exec cat {} \; | grep "image")
```

Comment:
@lulf not sure how it can work, because you don't read content of file

Reply:
Doh, fixed :)